src/lcm/simulate.py

```diff
@@ -445,7 +445,7 @@ def retrieve_non_sparse_choices(indices, grids, grid_shape):
     return out
 
 
-@partial(vmap_1d, variables=["index"])
+@partial(vmap_1d, variables=["index"], apply_allow_kwargs=False)
```

Comment:
```suggestion
@partial(vmap_1d, variables=["index"], callable_with="kwargs")
```
Argument will definitely have to be renamed in `vmap_1d`. This is just a suggestion -- instead of Boolean with an awkward name, use Literal that could be `args` or `kwargs`. But there could be better ideas, I did not think about it for very long.